detections/cloud/aws_saml_access_by_provider_user_and_principal.yml

```diff
@@ -0,0 +1,38 @@
+author: Rod Soto, Splunk
+date: '2021-01-26'
+description: This search provides specific SAML access from specific Service Provider,
+  user and targeted principal at AWS. This search provides specific information to
```

Comment:
@rosplk could you please clarify the context of a principal - is it a "windows" term, they use authentication/security principles as an umbrella term for anything that gets authenticated?